README.md

```diff
@@ -109,15 +109,15 @@ Instructions for implementing each feature follow.
         - Do `help history` in any `cmd2` application for more information
 
 - Simple scripting using ASCII text files with one command + arguments per line
```

Comment:
This should probably just say "text files" since utf-8 also works.

Reply:
Good catch